spec/features/request_options_spec.rb

```diff
@@ -97,7 +97,7 @@
       visit '/catalog/9741216'
     end
 
-    it 'does not display a request button', unless: in_travis? do
+    it 'does not display a request button', unless: in_travis? || in_circleci? do
```

Comment:
Can we just change all these to `in_ci?`?

Reply:
Yes I can. The in_circleci? checks for the ENV['circleci'] variable. I will define the in_ci? to check for the ENV['CI']. In this case I will try to remove the in_travis? from the spec.